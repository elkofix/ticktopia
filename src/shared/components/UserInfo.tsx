import { User } from "lucide-react"; 
export const UserInfo = ({ email }: { email?: string }) => (
  <div className="flex items-center">
    {/* Ícono solo visible en pantallas pequeñas */}
    <div className="flex sm:invisible items-center justify-center p-2 rounded-full bg-gray-200">
      <User className="h-5 w-5 text-gray-600" />
    </div>

  </div>
);
