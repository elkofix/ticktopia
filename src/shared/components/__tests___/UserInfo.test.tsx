import { render } from "@testing-library/react";
import { UserInfo } from "../UserInfo";

it('should match snapshot', () => {
  const { asFragment } = render(<UserInfo email="test@example.com" />);
  expect(asFragment()).toMatchSnapshot();
});