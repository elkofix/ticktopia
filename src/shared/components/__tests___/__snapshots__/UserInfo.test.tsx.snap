// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex items-center"
  >
    <div
      class="flex sm:invisible items-center justify-center p-2 rounded-full bg-gray-200"
    >
      <svg
        aria-hidden="true"
        class="lucide lucide-user h-5 w-5 text-gray-600"
        fill="none"
        height="24"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M19 21v-2a4 4 0 0 0-4-4H9a4 4 0 0 0-4 4v2"
        />
        <circle
          cx="12"
          cy="7"
          r="4"
        />
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
